clj-rn.conf.example.edn

```diff
@@ -0,0 +1,29 @@
+{:name "CljRnExample"
+
+ ;; dev builds from project.clj
+ :builds [{:id           :ios
+           :source-paths ["src" "env/dev"]
+           :figwheel     true
+           :compiler     {:output-to     "target/ios/app.js"
+                          :main          "env.ios.main"
+                          :output-dir    "target/ios"
+                          :optimizations :none}}
+          {:id           :android
+           :source-paths ["src" "env/dev"]
+           :figwheel     true
+           :compiler     {:output-to     "target/android/app.js"
+                          :main          "env.android.main"
+                          :output-dir    "target/android"
+                          :optimizations :none}}]
+
+ :js-modules ["realm"]
```

Comment:
Haven't found `js-modules` option in README. Not clear what is this option about and how to use it. Maybe it makes sense to add more information about possible config options to README.

Reply:
updated readme.